workers/unity/Packages/com.improbable.gdk.core/MutableView/MutableView.cs

```diff
@@ -285,18 +286,18 @@ internal void CreateEntity(long entityId)
                 return;
             }
 
-            var entity = entityManager.CreateEntity();
-            entityManager.AddComponentData(entity, new SpatialEntityId
+            var entity = EntityManager.CreateEntity();
+            EntityManager.AddComponentData(entity, new SpatialEntityId
             {
                 EntityId = entityId
             });
-            entityManager.AddComponentData(entity, new NewlyAddedSpatialOSEntity());
+            EntityManager.AddComponentData(entity, new NewlyAddedSpatialOSEntity());
 
-            addAllCommandRequestSenders(entity, entityId);
+            // addAllCommandRequestSenders(entity, entityId);
```

Comment:
Why is this commented out?

Reply:
Because it no longer exists - but its a reminder to replace that functionality when we rework commands

Reply:
would it better to add a todo & ticket number here to make sure it won't be forgotten

Reply:
Yes and no - the ticket doesn't exist yet because the design has not been written yet. Keep in mind the entire branch is a huge WIP as the main goal right now is to just stand something up with component updates and authority changes

This will almost certainly be hit when doing the worker abstraction and commands rework.